### PR TITLE
Feature/wmt 1007/national regional dist by grade

### PR DESCRIPTION
**Pull request by tsheppard01**

### CODE REVIEW

File: seed/views/01002_region_caseload_view.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
+exports.seed = function (knex, Promise) {
```
Comment (caoimheb) [inline seed/views/01002_region_caseload_view.js]: Can you merge in the latest `master`? The ordering on the naming of these files has changed

File: seed/views/01002_region_caseload_view.js [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+          , SUM(d1) AS d1
+          , SUM(c2) AS c2
+          , SUM(c1) AS c1
+          , SUM(b2) AS b2
+          , SUM(b1) AS b1
+          , SUM(a) AS a
+          , SUM(total_cases) AS total_cases
+        FROM app.ldu_caseload_view lv  
+          JOIN app.ldu l ON lv.id = l.id
+          JOIN app.region r ON l.region_id = r.id
```
Comment (caoimheb) [inline seed/views/01002_region_caseload_view.js]: Is this join to region needed? Can we just use `l.region_id`? Is there a foreign key constraint to ensure that `l.region_id` is valid?

Comment (caoimheb) [reply seed/views/01002_region_caseload_view.js]: Also in the team view

File: seed/data/dev/00200_offender_manager.js [hunk 3]
```diff
@@ -3,17 +3,20 @@ var tableName = 'offender_manager'
...
     .then(function (probationOfficerTypeId) {
-      // Inserts seed entries
-      return knex(tableName).insert([
-        { type_id: probationOfficerTypeId.id, key: 'JS01', forename: 'John', surname: 'Smith' },
-        { type_id: probationOfficerTypeId.id, key: 'JS02', forename: 'Tony', surname: 'Test' },
-        { type_id: probationOfficerTypeId.id, key: 'JS03', forename: 'Jane', surname: 'Doe' },
-        { type_id: probationOfficerTypeId.id, key: 'JS04', forename: 'Marcin', surname: 'Martin' },
-        { type_id: probationOfficerTypeId.id, key: 'JS05', forename: 'Courtney', surname: 'Larry' }
-      ])
+      return knex('offender_manager_type').select('id').where({description: 'Probation Support Officer'}).first()
```
Comment (caoimheb) [inline seed/data/dev/00200_offender_manager.js]: 👍

File: seed/views/01002_region_caseload_view.js (hunk 1 above)
Comment (tsheppard01) [reply seed/views/01002_region_caseload_view.js]: merged and renamed the new views

File: seed/views/01002_region_caseload_view.js (hunk 2 above)
Comment (tsheppard01) [reply seed/views/01002_region_caseload_view.js]: removed

File: seed/data/dev/00200_offender_manager.js (hunk 3 above)
Comment (tsheppard01) [reply seed/data/dev/00200_offender_manager.js]: thanks

**Pull request merged**